code/mathematical-algorithms/count_digits/count_digits.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import "fmt"
+
+func countDigits(n int) int {
```

Comment:
Could you fix the indent of this function?
Make all consistent.

Reply:
oops! Will do